### PR TITLE
x11-terms/terminator: support py3.10, use pep517

### PR DESCRIPTION
**Pull request by DarthGandalf**
Closes: https://bugs.gentoo.org/846359
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Alexey Sokolov <alexey+gentoo@asokolov.org>

### CODE REVIEW

**Comment by ConiKost** [comment]
While at it, could you also test with python3_11, if possible?

File: x11-terms/terminator/terminator-2.1.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{8,9,10} )
```
Comment (ConiKost) [inline x11-terms/terminator/terminator-2.1.1-r1.ebuild:7]: Please unifi syntax as `8..10`.

Comment (DarthGandalf) [reply x11-terms/terminator/terminator-2.1.1-r1.ebuild]: Done.

**Comment by DarthGandalf** [comment]
> While at it, could you also test with python3_11, if possible?

I can't, without adding lots of random python packages to accept_keywords

**Review by ConiKost — APPROVED** [review]

**Pull request closed without merging**